app/views/staff/reports/_tab.html.haml

```diff
@@ -0,0 +1,4 @@
+%li{class: "govuk-tabs__list-item #{ "govuk-tabs__list-item--selected" if active_tab == name }"}
+  = link_to t("tabs.report.#{name}"),
+    path,
+    { class: "govuk-tabs__tab", aria: { current: (active_tab == name ? "page" : nil) } }
```

Comment:
very interesting and neatly done! 

I had a look at the page on "Aditus" to which you linked -- and I see they have some tips on aria attributes for breadcrumbs -- which will probably come in useful when we add them.

https://www.aditus.io/patterns/breadcrumbs/

I've added a note on https://trello.com/c/2azNpzjB/1842-explore-how-breadcrumbs-could-aid-navigation

Reply:
Nice, good call. I learnt a lot about aria yesterday!